docs/en/observability/profiling-index-lifecycle-management.asciidoc

```diff
@@ -95,8 +93,11 @@ image::images/profiling-create-component-template.png[Create component template]
 [[profiling-ilm-custom-policy-rollover]]
 === Step 3: Rollover indices
 
-To confirm that Universal Profiling is now using the new index template and {ilm-init} policy, navigate to **{dev-tools-app}** and run the following:
+Confirm that Universal Profiling is now using the new index template and {ilm-init} policy:
 
+. To open **Console**, find `Dev Tools` in the {kibana-ref}/introduction.html#kibana-navigation-search[global search field].

```

Comment:
It feels weird starting a sentence with "To open xxx"  when the previous sentence doesn't talk about `xxx`.

Not sure how to rephrase it, because in the majority of places where we have `to open xxx` it feels natural.
Maybe something like:

```suggestion
. Open **Console** by finding `Dev Tools` in the {kibana-ref}/introduction.html#kibana-navigation-search[global search field].

```

Reply:
I totally agree with you here. I've tried to [follow the new guidelines](https://docs.elastic.dev/tech-writing-guidelines/ui-writing#referring-to-apps-and-pages) explicitly because a) it's not clear to me how proscriptive the guidelines are, and b) I wanted to make it easier to clean things up through search and replace if/when we decide to go with language that's clearer and more concise.

@mdbirnstiehl What are your thoughts here? Is it OK for us to change the sentence structure provided in the guidelines? (This is where examples in the guidelines would have been SUPER helpful.) The guidelines were updated after I started this PR and now say that we can adapt the patterns, but I'm not sure what that means.

Reply:
I give this my 🦭 of approval. Maybe we can mention to the style guide team that we could use some examples.